### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by JakeDaBoss**
- Changelog update.

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
Someone review please. @BlueNexus @Lykayos @WrongEnd @Nightspark43

File: html/changelogs/.all_changelog.yml [hunk 1]
```diff
@@ -3394,3 +3394,11 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - bugfix: Fixed three area's not having air alarms.
   - bugfix: Fixed Medical Examiner.
   - bugfix: Fixed upper Command power issues.
+2016-10-02:
+  N3X15:
```
Comment (BlueNexus) [inline html/changelogs/.all_changelog.yml]: I don't recall N3X15 working on this repo. Looks like _someone_ forgot to edit the name when they made their changelog. _looks at Oliver_

**Review by BlueNexus — CHANGES_REQUESTED** [review]

File: html/changelogs/.all_changelog.yml (hunk 1 above)
Comment (JakeDaBoss) [reply html/changelogs/.all_changelog.yml]: I had to make Lyka's, okay? :/

**Comment by JakeDaBoss** [comment]
Done @BlueNexus

**Review by BlueNexus — APPROVED** [review]

**Comment by BlueNexus** [comment]
I suggest asking #codershuttle about using qdel instead of del, before this is merged.

File: code/modules/admin/verbs/admincryo.dm [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+				var/obj/structure/closet/crate/secure/K = new /obj/structure/closet/crate/secure/(M.loc)
+				K.req_access += get_access(job)
+				K.name = (M.real_name + " - " + job + " - SSD Crate")
+				K.health = 1000000
+				K.contents = M.contents
+				for(var/datum/objective/O in all_objectives)
+					if(O.target && istype(O.target,/datum/mind))
+						if(O.target == M.mind)
+							if(O.owner && O.owner.current)
+								O.owner.current << "\red You get the feeling your target is no longer within your reach. Time for Plan [pick(list("A","B","C","D","X","Y","Z"))]..."
```
Comment (BlueNexus) [inline code/modules/admin/verbs/admincryo.dm]: Depreciated. Changed \red to `<span class="warning">`

File: code/modules/admin/verbs/admincryo.dm [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+					if ((T.fields["name"] == M.real_name))
+						del(T)
+				for(var/datum/data/record/G in data_core.general)
+					if ((G.fields["name"] == M.real_name))
+						del(G)
+
+				//Make an announcement and log the person entering storage.
+				//frozen_crew += "[M.real_name]"
+
+				message_admins("\blue [key_name_admin(usr)] has admin cryoed [key_name(M)]")
```
Comment (BlueNexus) [inline code/modules/admin/verbs/admincryo.dm]: Ditto, but for notice, not warning.

**Review by BlueNexus — CHANGES_REQUESTED** [review]

**Comment by JakeDaBoss** [comment]
I'll fix this this afternoon.

**Comment by JakeDaBoss** [comment]
@BlueNexus Always use qdel.

**Comment by BlueNexus** [comment]
Please correct all instances of del, then.

**Comment by JakeDaBoss** [comment]
Fixes #27

**Pull request closed without merging**